e2e/fixtures/.github/workflows/ci.yml

```diff
@@ -11,3 +11,8 @@ jobs:
     - uses: actions/setup-node
     # @OctoLinkerResolve(https://github.com/actions/upload-artifact)
     - uses: actions/upload-artifact@master
+    # @OctoLinkerResolve(https://github.com/github/codeql-action/tree/master/init)
+    - uses: github/codeql-action/init@v1
+    # @OctoLinkerResolve(https://github.com/OctoLinker/OctoLinker)
+    - uses: ./
```

Comment:
For some reason this test failed but it works correctly in the browser.

```
✕ resolves - uses: ./ to https://github.com/OctoLinker/OctoLinker (5001 ms)
```

Reply:
If I navigate to https://github.com/OctoLinker/OctoLinker/blob/936dbbc793bb8e2246a1ea7b6b34c76244dd28bd/e2e/fixtures/.github/workflows/ci.yml

this line links to https://github.com/OctoLinker/OctoLinker/tree/936dbbc793bb8e2246a1ea7b6b34c76244dd28bd/

which is inline with your implementation. Maybe we need another wildcard like `<root>` https://github.com/OctoLinker/OctoLinker/blob/master/e2e/generate-fixtures-json.js#L64

Reply:
How do you run the e2e tests locally? I adjusted `generate-fixtures-json.js` to point to my fork & branch, but I keep getting the following output when trying to run them:

```
❯ node "c:/dev/xt0rted/OctoLinker/node_modules/jest/bin/jest.js" "c:/dev/xt0rted/OctoLinker/e2e/automated.test.js" -t "End to End tests single blob"
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In C:\dev\xt0rted\OctoLinker
  198 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 37 matches
  testPathIgnorePatterns: C:\\dev\\xt0rted\\OctoLinker\\e2e\\ - 186 matches
  testRegex:  - 0 matches
Pattern: c:\\dev\\xt0rted\\OctoLinker\\e2e\\automated.test.js - 0 matches
```

The `fixtures.json` was created, but the tests in `automated.test.js` don't run.

Reply:
There is a special jest config for e2e tests `--config=e2e/jest-e2e.config.js` which is used when running `yarn e2e`